rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -336,6 +343,8 @@ void log_frame::CreateAndConnectActions()
 	{
 		QMenu* menu = m_log->createStandardContextMenu();
 		menu->addAction(m_clear_act);
+		menu->addAction(m_perform_goto_on_debugger);
+		m_perform_goto_on_debugger->setEnabled(m_log->textCursor().hasSelection());
```

Comment:
These should also check the condition
m_debugger_frame && m_debugger_frame->isVisible()

Reply:
Added.